pkg/game/field/field.go

```diff
@@ -119,16 +132,51 @@ func (field *Field) Expand(board elements.Board, cityManager city.Manager) {
 	field.features = newFeatures
 }
 
+// Returns true if the field has lesser or equal number of meeples than maxMeepleCount, and false otherwise
```

Comment:
While this description of the function is technically true, it does not necessarily make it clear how this is useful, i.e. it can be used to check whether a meeple could be placed on the given field.
With that use case in mind, the way to achieve this should probably also be documented since it's not necessarily clear what `maxMeepleCount` should be set to - this actually depends on whether the tile has any meeple on a field feature (always the case for the only usage - i.e. `fieldCanBePlaced` - which is why it needs to be passed with `1`).